api/api/views/media_views.py

```diff
@@ -272,7 +272,7 @@ def related(self, request, identifier=None, *_, **__):
             raise APIException(getattr(e, "message", str(e)))
         # If there are no hits in the search controller
         except IndexError:
-            raise APIException("Could not find items.", 404)
+            raise NotFound
```

Comment:
TIL raise accepts a class object rather than an instance for lazy evaluation of the parameterless constructor :open_mouth: https://docs.python.org/3/reference/simple_stmts.html#the-raise-statement

> It must be either a subclass or an instance of [BaseException](https://docs.python.org/3/library/exceptions.html#BaseException). If it is a class, the exception instance will be obtained when needed by instantiating the class with no arguments.

Reply:
Woah, TIL too!